pubspec.yaml

```diff
@@ -11,7 +11,7 @@ description: A mobile application for students at ACS UPB.
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 0.7.1+3
+version: 0.7.1+4
```

Comment:
That's the build number, not the patch version. See https://github.com/acs-upb-mobile/acs-upb-mobile/blob/master/CONTRIBUTING.md